proto/api/node/node.proto

```diff
@@ -35,31 +35,39 @@ message SvidUpdate {
 }
 
 // Represents a request to attest the node.
-message FetchBaseSVIDRequest {
+message AttestRequest {
     // A type which contains attestation data for specific platform.
-    spire.common.AttestedData attested_data = 1;
+    spire.common.AttestationData attestation_data = 1;
 
     // Certificate signing request.
     bytes csr = 2;
+
+    // Attestation challenge response
+    bytes response = 3;
 }
 
 // Represents a response that contains  map of signed SVIDs and an array of
 // all current Registration Entries which are relevant to the caller SPIFFE ID
-message FetchBaseSVIDResponse {
+message AttestResponse {
     // It includes a map of signed SVIDs and an array of all current
     // Registration Entries which are relevant to the caller SPIFFE ID.
     SvidUpdate svid_update = 1;
+
+    // This is a challenge issued by the server to the node. If populated, the
+    // node is expected to respond with another AttestRequest with the response.
+	// This field is mutually exclusive with the svid_update field.
```

Comment:
nit: spacing

Reply:
ahhh, tabs v.s spaces in proto files. looks like spaces. i'll update my editor config.